src/main/resources/assets/styles/home.less

```diff
@@ -27,7 +27,7 @@
 
   /* This is not used directly but required for processing by Webpack. DO NOT REMOVE! */
   #non-existing {
-    background-image: url('../images/background.jpg');
+    background-image: url('../images/background.webp');
```

Comment:
This definitely can be removed - even if comment states opposite- because we have e a copy webpack plugin that explicitly can handle this